next.config.js

```diff
@@ -1,5 +1,16 @@
 const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin");
 
+const self = "'self' https://*.replay.io wss://*.replay.io";
```

Comment:
Do we expect to run scripts from other subdomains? If not, I'd code to be specific with `https://app.replay.io` wherever possible.